napari/_vispy/_tests/test_vispy_multiscale.py

```diff
@@ -112,6 +112,10 @@ def test_multiscale_screenshot_zoomed(make_napari_viewer):
     target_center = np.array([255, 255, 255, 255], dtype='uint8')
     screen_offset = 3  # Offset is needed as our screenshots have black borders
 
+    # for whatever reason this is the only test where the border is 6px on hi DPI.
+    # if the 6 by 6 corner is all black assume we have a 6px border.
+    if not np.allclose(screenshot[:6, :6], np.array([0, 0, 0, 255])):
+        screen_offset = 6  # Hi DPI
```

Comment:
this seems like it's probably dependent on the amount of display scaling?

Reply:
Yes, but I don't know of any other scaling than x2. If we get to that point we can generalize. But I haven't found out how to get the scaling.